packages/app-form-builder/src/admin/components/FormEditor/Droppable.tsx

```diff
@@ -36,13 +40,19 @@ export interface IsVisibleCallable {
     * "ui" propetry gives us information about the Entity that we are moving.
     "Entity" can be step, field, row or custom. "Entity" will be custom in case we are moving field from a "Custom Field" menu.
     * "name" property contains the type of the field, it can be text, number or one of the available fields.
-    * "pos" propety contains info about Entity position that we are moving
+    * "pos" propety contains info about Entity position that we are moving.
```

Comment:
Can you double check this comment? Is it in line with the code? Like, it says `custom`, but it doesn't exist below.

Reply:
This still needs dbl checking.

Some of the things, like `"Entity" can be step, field, row or conditionGroup.` doesn't need a comment. We can see that in the TS type itself.

There are a couple of typos as well.

Finally, the comment and the actual type are not 100% in sync. Like, it says `pos` can be undefined, but in the type that's not true.

---

Let's again dbl check this comment. I guess we can delete this whole comment and only add a comment or two above the actual fields, if really needed.